### PR TITLE
Fix norms

### PR DESCRIPTION
**Pull request by thowell**
Removes `kRatioLoss` (was a duplicate of `kRectifyLoss`) and `kGeodesic` (not implemented). Importantly with the removal of `kGeodesic`, we keep the previous `NormType` numbering to ensure backward compatibility.

Minor: simplifies `kCosh` and `kPowerLoss` to use a single for loop.

### CODE REVIEW

File: mjpc/norm.cc [hunk 1]
```diff
@@ -14,11 +14,11 @@
 
 #include "mjpc/norm.h"
 
+#include <mujoco/mujoco.h>
```
Comment (erez-tom) [inline mjpc/norm.cc:17]: I'm not sure why your linter is switching the header order -- let's not do that.

**Review by erez-tom — COMMENTED** [review]
One small comment, otherwise LGTM.

File: mjpc/norm.cc (hunk 1 above)
Comment (thowell) [reply mjpc/norm.cc:17]: I think the formatter considers `<mujoco/mujoco.h>` to be a C system header. [Google C++ style: Names and Order of Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Comment (thowell) [reply mjpc/norm.cc]: reverted to the previous ordering

**Review by erez-tom — APPROVED** [review]

**Pull request merged**